### PR TITLE
Add generate_outputs stage

### PR DESCRIPTION
**Pull request by as2388**
Copied from OCHA, with references to OCHA renamed to WorldBank-PLR

### CODE REVIEW

File: generate_outputs.py [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+                                              target_file_name=messages_csv_drive_file_name,
+                                              target_folder_is_shared_with_me=True)
+
+        individuals_csv_drive_dir = os.path.dirname(pipeline_configuration.drive_upload.individuals_upload_path)
+        individuals_csv_drive_file_name = os.path.basename(pipeline_configuration.drive_upload.individuals_upload_path)
+        drive_client_wrapper.update_or_create(csv_by_individual_output_path, individuals_csv_drive_dir,
+                                              target_file_name=individuals_csv_drive_file_name,
+                                              target_folder_is_shared_with_me=True)
+
+        traced_data_drive_dir = os.path.dirname(pipeline_configuration.drive_upload.messages_traced_data_upload_path)
```
Comment (IsaackMwenda) [inline generate_outputs.py:173]: Hi @as2388 are we still uploading the traced data to drive? Not quite sure if we concluded on this. Feel free to raise this question on Tech & data.

File: generate_outputs.py [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+    with open(pipeline_configuration_file_path) as f:
+        pipeline_configuration = PipelineConfiguration.from_configuration_file(f)
+
+    if pipeline_configuration.drive_upload is not None:
+        log.info(f"Downloading Google Drive service account credentials...")
+        credentials_info = json.loads(google_cloud_utils.download_blob_to_string(
+            google_cloud_credentials_file_path, pipeline_configuration.drive_upload.drive_credentials_file_url))
+        drive_client_wrapper.init_client_from_info(credentials_info)
+
+    # Load the input datasets
```
Comment (IsaackMwenda) [inline generate_outputs.py:84]: I think we should move this function to the lib. What do you think?

**Review by IsaackMwenda — APPROVED** [review]

File: generate_outputs.py (hunk 2 above)
Comment (as2388) [reply generate_outputs.py:84]: agreed, thanks for the suggestion :)

Comment (as2388) [reply generate_outputs.py:84]: I'll do this in a future PR. Tracking this in #13.

File: generate_outputs.py (hunk 1 above)
Comment (as2388) [reply generate_outputs.py:173]: Following an offline discussion we'll remove this.

Comment (as2388) [reply generate_outputs.py:173]: PR for this opened at https://github.com/AfricasVoices/Project-WorldBank-PLR/pull/14

**Pull request merged**